src/main/scala/diplomacy/SRAM.scala

```diff
@@ -11,10 +11,11 @@ import freechips.rocketchip.util.DescribedSRAM
 abstract class DiplomaticSRAM(
     address: AddressSet,
     beatBytes: Int,
-    devName: Option[String])(implicit p: Parameters) extends LazyModule
+    devName: Option[String],
+    compatName: String = "sifive,sram0")(implicit p: Parameters) extends LazyModule
```

Comment:
I hate to do this, given that you have to pipe this through so many layers and across repositories, but I think it'd be a good idea to give this parameter a name that makes it very explicitly clear that this is meant to hold a DTS `compatible` string. For example, I'd prefer a name like `dtsCompatible`, or, if you find that too verbose, at the very least the fully spelled out `compatible`, since that's at least more obviously identifiable with the equivalent property in DTS.

Also, I think you should make this a `Seq[String]`, since this would allow one to set multiple compatible strings, as is allowed by the DTS spec.

Reply:
Ah yes, that is what I should do. I was having trouble making the dafault parameter in 2 places